fedimint-core/src/module/mod.rs

```diff
@@ -416,19 +511,18 @@ pub trait ModuleGen: Debug + Sized {
     /// to move from the previous database version to the current version.
     const DATABASE_VERSION: DatabaseVersion;
```

Comment:
At some point `ClientModuleGen` will also need to have Database Migrations right?

Reply:
Either way, I guess the `DATABASE_VERSION` shouldn't be shared

Reply:
Yes. We'll have to figure out the details with time. The exactly client modules will probably have to come from the new client code.